client/src/components/MagnifyingGlass/Button.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import './Button.css'
+import imageFile from '../../images/Search Icon_outlined.svg';
+
+const image = {
```

Comment:
What is this magnifying glass component? Why is the folder called `MagnifyingGlass` but the file is called `Button.js`?

Reply:
This is the one we closed as no longer necessary.